data/schema/index.ts

```diff
@@ -0,0 +1,27 @@
+// @flow
+/**
+ * This file provided by Facebook is for non-commercial testing and evaluation
+ * purposes only.  Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { GraphQLObjectType, GraphQLSchema } from "graphql";
+
+import { UserQuery } from "./queries/UserQuery";
+
+const Query = new GraphQLObjectType({
+  name: "Query",
+  fields: {
+    user: UserQuery,
+  },
+});
+
+export const schema = new GraphQLSchema({
```

Comment:
this is where we define the GraphQL server, the GraphQLXXX wrappers allow our script to build schema file from it.

then the schema file will be used by Relay to build typings